gulp/tasks/test.js

```diff
@@ -4,7 +4,6 @@ var gulp = require( 'gulp' );
 var $ = require( 'gulp-load-plugins' )();
```

Comment:
Yeah, there's this https://github.com/cfpb/cfgov-refresh/issues/830 open for that.